webcompat/helpers.py

```diff
@@ -495,7 +495,7 @@ def add_csp(response):
         "default-src 'self'; " +
         "object-src https://webcompat.github.io; " +
         "connect-src 'self' https://api.github.com; " +
-        "font-src 'self'; " +
+        "font-src 'self' https://fonts.gstatic.com/s/opensans/; " +
```

Comment:
Do we actually use opensans? I don't see it referenced anywhere.

Reply:
We do!
https://github.com/webcompat/webcompat.com/blob/issues/2011/webcompat/templates/layout.html#L34
https://github.com/webcompat/webcompat.com/blob/issues/2011/webcompat/static/css/src/typo.css#L2